index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Jose RM Portfolio</title>
+</head>
+
+<body id="mobile-design">
+  <header>
+    <nav class="navbar">
+      <p class="name-logo"><a href="#">Jose RM</a></p>
+      <ul class="navlinks">
+        <li class="links"><a href="#">Portfolio</a></li>
+        <li class="links"><a href="#">About</a></li>
+        <li class="links"><a href="#">Contact</a></li>
+        <li><a href="#"><img src="./images/e-mail-icon.png" alt="Email Send"></a></li>
+      </ul>
+      <div class="menu">
+        <span>&#9776</span>
+      </div>
+    </nav>
+  </header>
+
+  <section class="headline headline-desktop">
+
+    <div class="headline-info">
+      <h1 class="presentation-title">Hey There. <br>I’m Jose RM</h1>
+      <h2 class="what-i-do">I’m a Software Developer</h2>
+      <p class="support-text support-text-desktop">I can help you build a product, feature or website Look <br>
+        through
+        some of my work and experience! If you like what <br> you see and have a project you need coded,
+        <br> don’t hestiate to contact me.
+      </p>
+    </div>
+    <div class="lateral-bar">
+      <nav class="social-icons">
+        <a href="https://github.com/joramot" rel="noopener" target="_blank"><img src="./images/github-icon.png"
+            alt="GitHub Site Link"></a>
+        <a href="https://www.linkedin.com/in/jose-rm/" rel="noopener" target="_blank"><img
+            src="./images/linkedin-icon.png" alt="LinkedIn Social Media Link"></a>
+        <a href="#"><img src="./images/angelist-icon.png" alt="Angelist Social Media Link"></a>
+        <a href="https://twitter.com/el_rulis" rel="noopener" target="_blank"><img src="./images/twitter-icon.png"
+            alt="Twitter Social Media Link"></a>
+        <a href="#"><img src="./images/medium-icon.png" alt="Medium Social MediaLink"></a>
+      </nav>
```

Comment:
**OPTIONAL** - For semantic reasons, having two `nav` tags on a page is not advisable. I suggest you use a `ul` tag to wrap this section and wrap each social-icon in a corresponding `li` tag. Please fix this 🙏

Reply:
thanks so much @mckent05 I didn't know this, I learned new things today.